test/integration/node-specific/abort_signal.test.ts

```diff
@@ -70,17 +73,37 @@ describe('AbortSignal support', () => {
     await client.connect();
     db = client.db('abortSignal');
     collection = db.collection('support');
+
+    client.on('commandSucceeded', ev => {
+      if (
+        ev.commandName === 'find' ||
+        ev.commandName === 'aggregate' ||
+        ev.commandName === 'listCollections'
+      ) {
+        const cursorId = Long.isLong(ev.reply.cursor.id)
+          ? ev.reply.cursor.id
+          : Long.fromNumber(ev.reply.cursor.id);
+
+        if (!Long.ZERO.equals(cursorId)) {
+          cursors.push(cursorId);
```

Comment:
also - why is this file changed, if we don't update the tests?  Looks like all we're doing is tracking cursors.

Reply:
This is from command monitoring which, to prevent a backwards breaking change, deserializes the whole response with the user's BSON settings.

Reply:
How are cursors escaping these tests?  Client.close() already kills active cursors, so I'd expect that:

1. any test that closes its client also cleans up all cursors created for the test
2. all tests close their clients

So, we'd have no leaking cursors as a result.